cli/lib/kontena/cli/nodes/list_command.rb

```diff
@@ -19,10 +19,19 @@ def node_name(node, grid)
     end
 
     def node_status(node)
-      if node['connected']
-        node['availability'] == 'drain' ? pastel.yellow('drain') : pastel.green('online')
+      case node_status = node['status']
+      when 'created'
+        "#{pastel.dark('created')} #{time_since(node['created_at'], terse: true)}"
+      when 'connecting'
+        "#{pastel.cyan('connecting')} #{time_since(node['connected_at'], terse: true)}"
+      when 'online'
+        "#{pastel.green('online')} #{time_since(node['connected_at'], terse: true)}"
+      when 'drain'
+        "#{pastel.yellow('drain')} #{time_since(node['connected_at'], terse: true)}"
```

Comment:
Not really, there's no timestamp for how long the node has been draining?

Fixed to just show `drain` without any duration.